tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -716,10 +716,6 @@ qos/test_qos_sai.py::TestQosSai::testQosSaiDwrrWeightChange:
       - "asic_type in ['mellanox']"
 
 qos/test_qos_sai.py::TestQosSai::testQosSaiHeadroomPoolSize:
-  xfail:
-    reason: "Image issue on Arista platforms"
-    conditions:
-      - "platform in ['x86_64-arista_7050cx3_32s']"
   skip:
     reason: "Headroom pool size not supported."
     conditions:
```

Comment:
@XuChen-MSFT I see this case supports the following hwsku, if not added them to skip while list, it will still be skipped. I think we have to add these 2 hwsku in the skip condition.
```
        'Arista-7050CX3-32S-C32',
        'Arista-7050CX3-32S-D48C8
```'

Reply:
resolved, added them to support lit